lib/karafka/connection/client.rb

```diff
@@ -305,6 +305,8 @@ def internal_commit_offsets(async: true)
         case e.code
         when :assignment_lost
           return false
+        when :unknown_member_id
+          return false
```

Comment:
I guess `internal_store_offset` can not cause an error with this code since it does not contact the broker directly, and that's why you're not rescuing it there? I'm just double checking since `assignment_lost` is rescued both here and in `internal_store_offset`, and I'm not that knowledgable of librdkafka internals.

Reply:
Yes. Exactly.